### PR TITLE
Script to prepare binary executables for uploading to the github release page.

### PR DESCRIPTION
**Pull request by Unisay**
- [x] `./scripts/prepare-bins.sh`
- [x] Tweak RELEASE.adoc.

### CODE REVIEW

File: RELEASE.adoc [hunk 1]
```diff
@@ -75,19 +75,11 @@ This updates versions and version bounds, and assembles the changelogs.open a PR
...
 - Click `Publish release`.
-7. Open a PR in the https://github.com/IntersectMBO/cardano-haskell-packages[CHaP repository] for publishing the new version. Run `./scripts/add-from-github.sh "https://github.com/IntersectMBO/plutus" COMMIT-SHA LIST-OF-UPDATED-PACKAGES` (see https://github.com/IntersectMBO/cardano-haskell-packages#-from-github[the README on CHaP]). Example: https://github.com/IntersectMBO/cardano-haskell-packages/pull/394.
+7. Open a PR in the https://github.com/IntersectMBO/cardano-haskell-packages[CHaP repository] for publishing the new version. +
+If you are making PR from your own fork then don't forget to sync your fork with the upstream first. +
+Run `./scripts/add-from-github.sh "https://github.com/IntersectMBO/plutus" COMMIT-SHA LIST-OF-UPDATED-PACKAGES` (see https://github.com/IntersectMBO/cardano-haskell-packages#-from-github[the README on CHaP]). Example: https://github.com/IntersectMBO/cardano-haskell-packages/pull/764.
```
Comment (Unisay) [inline RELEASE.adoc:82]: I've sneakily changed the example PR to a newer one with a slightly better description.

**Review by zliu41 — APPROVED** [review]
:+1:

**Comment by bezirg** [comment]
This would require that `upx` is already installed in the developer's environment (outside nix). In my case:

```
$upx
bash: upx: command not found
```

I think it would be better to do it inside nix itself. 
But this PR is already better than what we had befoer, so no objections to that.

**Comment by Unisay** [comment]
> This would require that upx is already installed in the developer's environment (outside nix). 

I don't think so! In the shebang I used `nix-shell` to make sure the `upx` package is available, see `-p upx`.

**Comment by bezirg** [comment]
> > This would require that upx is already installed in the developer's environment (outside nix).
> 
> I don't think so! In the shebang I used `nix-shell` to make sure the `upx` package is available, see `-p upx`.

Oups sorry, i didn't see the shebang :(

**Pull request merged**